wagtail/contrib/redirects/utils.py

```diff
@@ -45,5 +47,10 @@ def get_file_storage():
     raise Exception("Invalid file storage, must be either 'tmp_file' or 'cache'")
 
 
+def get_redirect_hash(url):
+    namespace = "4ccf35b6-0e63-49ec-a878-50e680f3ecfc"
```

Comment:
Non-essential suggestion: I know we've probably done similar things in a few places, but this might be better off as a global, to make it clearer / easier to access.

Reply:
Done